src/hooks/feature-cards/spindl/useSpindlCards.ts

```diff
@@ -0,0 +1,85 @@
+import { useAccount } from '@lifi/wallet-management';
+import {
+  useWidgetEvents,
+  WidgetEvent,
+  type RouteExecutionUpdate,
+} from '@lifi/widget';
+import { useCallback, useEffect } from 'react';
+import { useCallRequest } from 'src/hooks/useCallRequest';
+import type { SpindlFetchData, SpindlFetchParams } from 'src/types/spindl';
+import { callRequest } from 'src/utils/callRequest';
+import { getLocale } from 'src/utils/getLocale';
+import { getSpindlConfig } from './spindlConfig';
+import { useSpindlProcessData } from './useSpindlProcessData';
+
+const FETCH_SPINDL_PATH = '/render/jumper';
+
+function isSpindlFetchData(data: unknown): data is SpindlFetchData {
+  return (
+    typeof data === 'object' &&
```

Comment:
same comment as of https://github.com/jumperexchange/jumper-exchange/pull/1659/files#diff-d2ed522d9d87e46a85375bf9b06b5deae36b2a1bfc129670a932e319ddc31c9dR124